src/index.css

```diff
@@ -113,10 +113,15 @@ code {
 }
 
 .camera-button-icon-container {
-  width: 32px;
-  height: 32px;
+  width: 16px;
+  height: 16px;
   padding-bottom: 5px;
   display: flex;
   justify-content: center;
   align-items: center;
 }
+
+img{
```

Comment:
Since you're setting the width here, you don't need to set it up above for `.camera-button-icon-container`